gaia-header.js

```diff
@@ -123,6 +124,7 @@ module.exports = component.register('gaia-header', {
     debug('attached');
     this.runFontFitSoon();
     this.observerStart();
+    window.addEventListener('resize', this);
```

Comment:
I'd still prefer a `window.addEventListener('resize', this.onResize);`, it reads clearer to me and negates the need for the `switch` function junk in `.handleEvent` :)

Although this will require something like: `this.onResize = this.onResize.bind(this)` to maintain context and facilitate `removeEventListener()`.